### PR TITLE
Add comment functions for JS and Java

### PR DESCRIPTION
**Pull request by emejia01**
These commits consist of changes to the script.js file and servlet file for the comment system functionality. Some changes to the html file were also made but more modifications are coming to make it more responsive.

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -19,14 +19,52 @@
...
 import javax.servlet.http.HttpServletResponse;
+// add imports
+import java.util.ArrayList;
+import com.google.gson.Gson;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+    ArrayList<String> messages = new ArrayList<String>();// {
+    //    {
```
Comment (qirongwang) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:30]: remove these comments?

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 2]
```diff
@@ -19,14 +19,52 @@
...
+    //};
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+
     response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello Erik!</h1>");
+    //response.getWriter().println("<h1>Hello Erik!</h1>");
+
+    //messages.add("hello");
```
Comment (qirongwang) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:43]: same here

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 3]
```diff
@@ -19,14 +19,52 @@
...
+
+    Gson gson = new Gson();
+    String jsonString = gson.toJson(messages);
+    response.getWriter().println(jsonString);
+
+    
+  }
+
+    @Override
+    public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
```
Comment (qirongwang) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:55]: nit: indent of this function seems off

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 1 above)
Comment (emejia01) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:30]: Will do.I forgot to remove them after debugging.

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 4]
```diff
@@ -19,14 +19,52 @@
...
+        String comment_text = getParameter(request, "text-input", "");
```
Comment (qirongwang) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:56]: Should we store a default value as empty string or just don't store? I feel we should log error when value is null and not store an empty comments

**Pull request merged**